### DISCUSSION TITLE
How much do you want to know about Main branch issues?

### DISCUSSION THREAD

**Discussion by 9mm** [neovide/neovide#2019] [General]
Im trying out `--HEAD` from homebrew to help with the render loop branch and just live on the edge.

I see some things but I don't want to make issues for them as obviously it's in a complete state of development and it will create tons of noise for things probably already everyone knows about, however, maybe they're also helpful. I'm not sure.

I noticed file descriptors dont seem to close on OSX.. as after I use Neovide a few hours I start getting all kinds of crazy warnings about watching files, vue/vite stops watching files, swap files go crazy. It is as if files are not being closed

**Comment by fredizzimo** [comment]
For Neovide main, you should report everything. We consider that potentially as stable as the releases, but there's less testing, so it might still have bugs, that we would like to know of as soon as possible. In practice we will create a release when there has been no critical bug reports related to it, and there are either critical features or bug fixes that we want to release. It also helps if you test the latest release, so we get a better idea of when something might have broke.

For PRs like the render loop one, but also for smaller ones, when they are out of draft, they should be considered stable. And any bugs you find related to them should be reported there, so that it can be judged if it needs to be fixed before merging, or if it's less critical and can be fixed later. Obviously for these types of bugs, you should ensure that the bug is unique to the PR in question before reporting it there, otherwise it should be a regular issue.

If you are running anything else than the latest Neovim release in combination with Neovide, it helps if you test with the latest release to see if the issue occurs there as well, and also if it occurs in the terminal mode of Neovim, so that you don't report Neovim bugs as Neovide ones. Even if it's a Neovide specific one, but only occurs in the nightly, it's still worth reporting here, since the nightly will eventually end up being released, so we need to get it fixed either here or in Neovim.

In fact, I believe you are running into  Neovim bug right now, this issue https://github.com/neovim/neovim/issues/23291. The easiest solution is to downgrade to Neovim 0.9.2, which is now released and works with the rendering branch.

And always try to search as well as you can for both open and closed issues in both repositories. I know that it can be hard to know what to search for, so it's ok with some duplicates, but it saves everyone's time if you do so.

Questions are generally better to be asked here in the discussions section, to not clutter up PRs and issues too much. Or alternatively in our discord or matrix channels, depending on what you ask.

**Comment by 9mm** [comment]
Ok, i have downgraded and this is super helpful info. Great tip on checking nvim first, i will do that as well.
